Marksman/Refs.fs

```diff
@@ -305,6 +322,7 @@ module Dest =
             | H h when Heading.isTitle h.data ->
                 [| Dest.Doc { link = h.text; kind = FileLinkKind.Title; dest = srcDoc } |]
             | H h -> [| Dest.Heading(Implicit srcDoc, h) |]
+            | T t -> findTagOccurrences folder t.data
```

Comment:
This is a bit off – `declsToFind` is a list of things that you want to find references to, so in this case it might be
```
| T t -> [| Dest.Tag(srcDoc, t) |]
```